clients/coredata/event.go

```diff
@@ -149,6 +154,10 @@ func (e *eventRestClient) Add(event *models.Event, ctx context.Context) (string,
 	}
 }
 
+func (e *eventRestClient) AddBytes(event []byte, ctx context.Context) (string, error) {
```

Comment:
Note that responsibility is on the caller to specify context (esp content type) for AddBytes method.